third_party/vmServiceDrivers/org/dartlang/vm/service/VmService.java

```diff
@@ -226,30 +228,25 @@ public void evaluateInFrame(String isolateId, int frameIndex, String expression,
   }
 
   /**
-   * Valid values for [gc] are 'full'.
-   *
-   * @undocumented
+   * The [getAllocationProfile] RPC is used to retrieve allocation information for a given isolate.
    */
   public void getAllocationProfile(String isolateId, AllocationProfileConsumer consumer) {
     final JsonObject params = new JsonObject();
     params.addProperty("isolateId", isolateId);
-    request("_getAllocationProfile", params, consumer);
+    request("getAllocationProfile", params, consumer);
   }
 
   /**
-   * Valid values for [gc] are 'full'.
-   *
-   * @undocumented
-   *
-   * @param gc This parameter is optional and may be null.
+   * The [getAllocationProfile] RPC is used to retrieve allocation information for a given isolate.
    * @param reset This parameter is optional and may be null.
+   * @param gc This parameter is optional and may be null.
```

Comment:
Aside (and maybe for another time): consider annotation these `@Nullable`?

Reply:
This library is generated by another tool (https://github.com/dart-lang/vm_service_drivers) - it would be tricky to reference `org.jetbrains.annotations` classes.

Reply:
Actually, to follow up on this, I think that IntelliJ's NNBD code also works with the `javax.annotation.Nullable` / `javax.annotation.Notnull` annotations, so we can add NNBD metadata to these calls. Definitely worth looking into.

Reply:
https://github.com/dart-lang/vm_service_drivers/pull/250, thought I don't know if its something we should land